dependencies.md

```diff
@@ -2,14 +2,16 @@
 
 This file lists the dependencies used in this repository.
 
-| Dependency                                       | License                                 |
-|--------------------------------------------------|-----------------------------------------|
-| github.com/antonmedv/expr                        | MIT License                             |
-| github.com/dustin/go-humanize                    | MIT License                             |
-| github.com/google/go-cmp                         | BSD 3-Clause "New" or "Revised" License |
-| github.com/klauspost/compress                    | MIT License                             |
-| github.com/nats-io/nats-server                   | Apache License 2.0                      |
-| github.com/nats-io/nats.go                       | Apache License 2.0                      |
-| golang.org/x/net                                 | BSD 3-Clause "New" or "Revised" License |
-| golang.org/x/text                                | BSD 3-Clause "New" or "Revised" License |
-| gopkg.in/yaml.v3                                 | Apache License 2.0                      |
+| Dependency                        | License      |
+|-----------------------------------|--------------|
+| github.com/antonmedv/expr         | MIT          |
+| github.com/dustin/go-humanize     | MIT          |
+| github.com/google/go-cmp/cmp      | BSD-3-Clause |
+| github.com/klauspost/compress     | Apache-2.0   |
```

Comment:
Actually this is wrong, looking at the repo he has a bunch of licences in the one file and the MIT one is first and applies to the main repo, I think it was right before @piotrpio

Reply:
Actually, I am not even sure what that license is, but its not ASL-2 or MIT

Reply:
It's `BSD-3-Clause` I guess, same as go